src/test/scala/is/hail/methods/DeNovoSuite.scala

```diff
@@ -0,0 +1,157 @@
+package is.hail.methods
+
+import is.hail.SparkSuite
+import is.hail.check._
+import is.hail.expr._
+import is.hail.utils._
+import is.hail.variant.{GenericGenotype, Genotype, VSMSubgen, Variant, VariantDataset, VariantSampleMatrix}
+import org.apache.commons.math3.distribution.{BinomialDistribution, NormalDistribution}
+import org.apache.spark.sql.Row
+import org.testng.annotations.Test
+
+import scala.sys.process._
+
+class DeNovoSuite extends SparkSuite {
+  val gen: Gen[(VariantDataset, Pedigree)] = {
```

Comment:
add a couple lines of comment explaining your testing approach (generate random data, run through Hail and the Kaitlin's Python program, compare results...)